src/devtools/views/Components/Element.css

```diff
@@ -73,3 +73,13 @@
   height: 1rem;
   color: var(--color-expand-collapse-toggle);
 }
+
+.Badge{
+  background: var(--color-component-name);
+  color: #000;
+  line-height: calc(var(--line-height-data) - 0.25rem);
+  padding: 0 0.55rem;
```

Comment:
We should stick to increments of `0.25` (or `0.125` in a pinch) like we do in other places, so the UI feels more consistent.